website/docs/kubernetes/kind.md

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 1
+title: Kind support
+description: Kind is one way to get Kubernetes running on your workstation.
+keywords: [podman desktop, podman, containers, migrating, kubernetes, kind]
+tags: [migrating-to-kubernetes, kind]
+---
+
+# Running Kubernetes on your workstation with Kind
+
+Kind is a command line tool that can create Kubernetes clusters on your favorite container engine.
+
+It has experimental support for Podman.
+However, Kind has specific requirements that need configuration tuning.
+
+## Running Kind on Windows Subsystem for Linux (WSL)
+
+Due to incompatibilities between WSL and systemd, Kind does not work with the `rootless` mode.
+Therefore, to use Kind with your Podman machine, configure `rootful` mode.
+
+### Configuring an existing Podman machine to run Kind
+
+When you create a Podman machine, Podman creates two system connections:
+
+* `rootless`
+* `rootful`
+
+Kind use the default Podman connection.
+Therefore, you must set the default connection to `rootful`.
+
+#### Procedure
+
+1. List the Podman system connections:
+
+  ```shell
+  > podman system connection ls
+  
+You should see a similar output:
+  ```shell
+  Name                         URI                                                          Identity                                   Default
+  podman-machine-default       ssh://user@localhost:54133/run/user/1000/podman/podman.sock  C:\Users\Jeff\.ssh\podman-machine-default  true
+  podman-machine-default-root  ssh://root@localhost:54133/run/podman/podman.sock            C:\Users\Jeff\.ssh\podman-machine-default  false
+  ```
+
+2. Modify the default connection to be rootful
+
+The rootful connection is suffixed by `-root`. If it is not the default one, you must issue the following command:
+
+  ```shell
+  > podman system connection default podman-machine-default-root
+  ```
+
+### Creating a Podman machine ready to run Kind
+
+#### Prerequisites
+
+* No existing Podman machine 
+
+#### Procedure
+
+* Create a rootful Podman machine:
+
+  ```shell
+  podman machine init --rootful my-machine-name
+
```

Comment:
(missing enf of block)

```suggestion
    ```

```

Reply:
Does removing the line following really fix the issue